presto-main/src/test/java/com/facebook/presto/type/TestArrayOperators.java

```diff
@@ -328,6 +328,20 @@ public void testSort()
     }
 
     @Test
+    public void testArrayIntersect()
+            throws Exception
+    {
+        assertFunction("ARRAY_INTERSECT(ARRAY [12], ARRAY [10])", Lists.newArrayList());
+        assertFunction("ARRAY_INTERSECT(ARRAY['foo', 'bar', 'baz'] , ARRAY['foo', 'test', 'bar'])", ImmutableList.of("bar", "foo"));
```

Comment:
add a space after "ARRAY" to match other tests in this file, and no space after ","